covid19-etl/etl/dsfsi.py

```diff
@@ -79,21 +79,161 @@ def normalize_symptoms(symptoms):
     return result
 
 
-def normalize_date(d):
-    if not d:
+def normalize_date(date):
+    if not date or date in ["NA", "N/A"]:
         return None
 
+    d = date
+    d = d.replace("Returned", "")
+    d = d.replace("between", "")
+    d = d.strip()
```

Comment:
DSFSI is the worst dataset if compared by effort/usefulness. 😅

Reply:
yes 🤦‍♀️